src/content/guides/development.md

```diff
@@ -173,14 +170,13 @@ Let's add an npm script that will start webpack's Watch Mode:
  }
 ```
 
-Tell `CleanWebpackPlugin` that we don't want to remove the `index.html` file after the incremental build triggered by watch. We do this with the [`cleanStaleWebpackAssets` option](https://github.com/johnagan/clean-webpack-plugin#options-and-defaults-optional):
+Tell `webpack` that we don't want to remove the `index.html` file after the incremental build triggered by watch. We do this with the [`output.clean`](/configuration/output/#outputclean):
```

Comment:
`index.html` is not removed running `webpack --watch` with `clean: true` in my test. Can you double check it?

Reply:
Yeah, `index.html` is not removed. Updated 👍🏻